lib/src/screens/setup_2fa/setup_2fa_qr_page.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:cake_wallet/generated/i18n.dart';
+import 'package:cake_wallet/routes.dart';
+import 'package:cake_wallet/src/screens/base_page.dart';
+import 'package:cake_wallet/src/screens/receive/widgets/qr_image.dart';
+import 'package:cake_wallet/utils/show_bar.dart';
+import 'package:cake_wallet/view_model/set_up_2fa_viewmodel.dart';
+
+import '../../../palette.dart';
+import '../../widgets/primary_button.dart';
+import '../../widgets/standard_list.dart';
+
+class Setup2FAQRPage extends BasePage {
+  Setup2FAQRPage({required this.setup2FAViewModel});
+
+  final Setup2FAViewModel setup2FAViewModel;
+
+  @override
+  String get title => 'Set up Cake 2FA';
+
+  @override
+  Widget body(BuildContext context) {
+    final totpLink =
+        'otpauth://totp/Cake%20Wallet:${setup2FAViewModel.deviceName}?secret=${setup2FAViewModel.secretKey}&issuer=Cake%20Wallet&algorithm=SHA1&digits=6&period=30';
```

Comment:
For compatibility purposes, let's save a value somewhere in the wallet and backup that specifies the TOTP version. We can specify version 1 as this standard, and we can later change a version to to use SHA512 when it's more adopted, for example.

Reply:
Moved this